README.md

```diff
@@ -40,28 +62,37 @@ if ((n - Math.floor(n)) >= .5) {
   i = Math.floor(n);
 }
 ```
+//my solution
 
+var i = Math.round(n)
 
-**b)**
+**b)** - not done
 ```
 var y, xIsFalse = (x? false : true);
 if (xIsFalse)
    y = false;
 else
    y = x;
+
```

Comment:
This one took me a while as well. I find its best to break thinks apart line by line in the console. when I find code I can't read, I cut and paste each line one at a  time into the console; change the variables, play with it until I think I understand what it does. Only then will I move to the next line. If you progress through the function like this you will have a esier time figuring out whats going on. Also xIsFalse is a very confusing variable name, since x COULD be true.